### PR TITLE
Npm tests + npm cli workdir calculation bug fix

### PR DESCRIPTION
**Pull request by barbelity**

### CODE REVIEW

File: tests/tests.js [hunk 1]
```diff
@@ -81,27 +87,41 @@ describe("JFrog Artifactory VSTS Extension Tests", () => {
                 default:
                     assert.fail("Unsupported OS found: " + os.type());
             }
-        })
+        });
+    });
+
+    describe("Unit Tests", () => {
```
Comment (yahavi) [inline tests/tests.js]: Please move all "Utils tests" to "Unit tests"

File: tests/resources/npm/resources/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "description": "Npm install test",
+  "dependencies": {
+    "fs-extra": "^7.0.0",
+    "request-promise-lite": "^0.13.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```
Comment (yahavi) [inline tests/resources/npm/resources/package.json]: "author": "JFrog",
"license": "Apache-2.0"

File: tests/resources/npm/files/package.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "description": "Npm install test",
+  "dependencies": {
+    "fs-extra": "^7.0.0",
+    "request-promise-lite": "^0.13.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```
Comment (yahavi) [inline tests/resources/npm/files/package.json]: "author": "JFrog",
"license": "Apache-2.0"

File: tests/resources/npm/files/jfrog-npm-install-test-1.0.0.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "author" : "",
+  "dist" : {
+    "tarball" : "http://localhost:8081/artifactory//vsts-npm-local-test/jfrog-npm-install-test/-/jfrog-npm-install-test-1.0.0.tgz",
+    "shasum" : "77fd31c38b8b98001e3ef4de5b90fadc1d0d2289"
+  },
+  "dependencies" : {
+    "fs-extra" : "^7.0.0",
+    "request-promise-lite" : "^0.13.1"
+  },
+  "license" : "ISC",
```
Comment (yahavi) [inline tests/resources/npm/files/jfrog-npm-install-test-1.0.0.json]: "author": "JFrog",
"license": "Apache-2.0"

Comment (barbelity) [reply tests/resources/npm/files/jfrog-npm-install-test-1.0.0.json]: fixed

File: tests/resources/npm/files/package.json (hunk 3 above)
Comment (barbelity) [reply tests/resources/npm/files/package.json]: fixed

File: tests/resources/npm/resources/package.json (hunk 2 above)
Comment (barbelity) [reply tests/resources/npm/resources/package.json]: fixed

File: tests/tests.js (hunk 1 above)
Comment (barbelity) [reply tests/tests.js]: done

**Pull request merged**